assets/js/polyfills.js

```diff
@@ -3,6 +3,8 @@ import 'regenerator-runtime/runtime';
 import 'whatwg-fetch';
 import objectFitImages from 'object-fit-images';
 
+require('formdata-polyfill');
+
```

Comment:
Any reasons to use require instead of import?

Reply:
it's mentioned in polyfill docs.

Reply:
Could you try import instead of require?